src/datamigration/dags/datamigration_utils/hive_bq_load_utils_inc.py

```diff
@@ -462,13 +462,15 @@ def get_job_status(tbl, result):
     """
     if result.returncode == 0:
         print("Loaded table: {} ".format(tbl))
-        print("BigQuery job id: {}".format(result.stderr.split(" ")[2]))
+        # print("BigQuery job id: {}".format(result.stderr.split(" ")[2]))
```

Comment:
@dmedora Good catch! Removed it in the new commit. 
FYI - There was little error that we discovered with testing. 
The print command makes it mandatory for us to have result.stderr.split(" ")[2] else it errors out instead of going to the else in the following logic.
It's just a print statement so no major impact but I have moved the print statement to if and if else below.